Gulpfile.js

```diff
@@ -5,7 +5,6 @@ const through = require("through2");
 const chalk = require("chalk");
 const newer = require("gulp-newer");
 const babel = require("gulp-babel");
-const watch = require("gulp-watch");
 const gutil = require("gulp-util");
```

Comment:
I can make a new PR with only the change that you've made to the Gulpfile, would you mind? It seems to me that theses changes are not really related.